src/main/java/graphql/introspection/Introspection.java

```diff
@@ -8,6 +8,7 @@
 import graphql.PublicApi;
```

Comment:
The tests pick up the fields in place - we have great coverage here.

I put a debug point in `PropertyDataFetcher` when running the main tests and I never hit it for introspection related calls - so one more reason on why its not all covered by actual DFs